docs/modules/supernova.md

```diff
@@ -0,0 +1,24 @@
+---
+Name: Supernova
+URL: https://www.10xgenomics.com/
+Description: >
+    Supernova is a de novo genome assembler for 10X Genomics linked-reads.
+---
+
+## Important notes
```

Comment:
Please make these h4 headings (`####`) for consistency with other module docs - thanks!